src/frontend/main/CityTables.tsx

```diff
@@ -0,0 +1,222 @@
+    width: auto;
+  }
+
+  th:nth-child(3),
+  td:nth-child(3) {
+    padding-right: 16px;
+  }
+
+  td {
+    text-align: center;
+  }
+
+  tbody tr:nth-child(1) {
+    th,
+    td {
+      padding-top: 8px;
+    }
+  }
+`;
+
+const TableContainer = styled.div`
+  width: 100%;
+  border-bottom: 1px solid ${props => props.theme.grey};
+  padding: 0 0 24px 0;
+
+  p {
+    margin: 0 0 4px 0;
+  }
+`;
+
+const CityHeadingContainer = styled.div`
+  padding: 0 16px;
+`;
+
+const BoldText = styled.p`
+  font-weight: bold;
+`;
+
+const ItalicText = styled.p`
+  font-style: italic;
+`;
+
+const CityHeading = styled.h2`
+  font-size: 21px;
+  margin: 18px 0 8px;
+`;
+
+const TableHead = styled.thead`
+  background: ${props => props.theme.lightGrey};
+
+  th {
+    padding: 6px 4px;
+    font-style: italic;
+    font-size: 14px;
+  }
+`;
+
+const Loading = styled.p`
+  margin-top: 40px;
+`;
+
+const TableViewWrapper = styled.div<TableViewWrapperProps>`
+  max-width: 600px;
+  margin: 0 auto;
+  padding-bottom: 40px;
+  height: ${props => (props.isEmbed ? `calc(100vh - (${CITYSELECTHEIGHT}px + ${NAVHEIGHT}px))` : 'auto')};
+  overflow: ${props => (props.isEmbed ? 'auto' : 'initial')};
+`;
+
+const CityTables = ({ data, selectedCity, isEmbed }: CityTablesProps) => {
+  const [listItems, setListItems] = useState(data.slice(0, 10));
+  const [isFetching, setIsFetching] = useState(false);
+  const container = useRef<HTMLDivElement>(null);
+
+  const fetchMoreListItems = useCallback(() => {
+    if (isFetching) return;
+    setIsFetching(true);
+
+    setTimeout(() => {
+      setListItems((prevState: any) => [...prevState, ...data.slice(prevState.length, prevState.length + 10)]);
+      setIsFetching(false);
+    }, 2000);
+  }, [data, isFetching]);
+
+  // For the main site, where the page is scrolled
+  const handleWindowScroll = useCallback(() => {
+    if (!isEmbed && selectedCity === '' && data.length > listItems.length) {
+      if (window.innerHeight + document.documentElement.scrollTop !== document.documentElement.offsetHeight) return;
+      fetchMoreListItems();
+    }
+  }, [isEmbed, selectedCity, data, fetchMoreListItems, listItems]);
+
+  // For article embeds, where the the container is scrolled
+  const handleContainerScroll = () => {
+    if (isEmbed && container.current !== null && selectedCity === '' && data.length > listItems.length) {
+      var scrollY = container.current.scrollHeight - container.current.scrollTop;
+      var height = container.current.offsetHeight;
+      var offset = height - scrollY;
+
+      if (offset === 0 || offset === 1) {
+        fetchMoreListItems();
+      }
+    }
+  };
+
+  useEffect(() => {
+    const cityList =
+      selectedCity === ''
+        ? data
+        : data.filter((item: any) => {
+            return item.city === selectedCity;
+          });
+    setListItems(cityList.slice(0, 10));
+  }, [data, selectedCity]);
+
+  useEffect(() => {
+    window.addEventListener('scroll', handleWindowScroll);
```

Comment:
This means the `handleWindowScroll` handler is still listening to window scroll events in embed mode. Would it be a bit more efficient to only addEventListener when not in embed?

Reply:
....aalthough I'm not sure how window on scroll would behave in iframe though :thinking:  :D

Reply:
Hmm true maybe the isEmbed check should be here instead